### PR TITLE
single linters: Add regex linter.

### PR DESCRIPTION
**Pull request by jiviteshjain**
### Sample output:
image

### Checks for
- Commit title against a regex
- Length of the commit title
- Length of each line in the commit body

### Note:
- `LinterLevel` should inherit `IntEnum` to allow comparisons between levels.

### CODE REVIEW

File: git_lint_branch/linter_output.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-from enum import Enum
+from enum import IntEnum
 
-class LinterLevel(Enum):
+class LinterLevel(IntEnum):
```
Comment (juped) [inline git_lint_branch/linter_output.py:3]: This makes sense to do. Could you break it out into a different commit in the history, though? (Doesn't need to be a whole different branch, IMO.)

**Review by juped — CHANGES_REQUESTED** [review]
Looks sensible. We'll go back and wire it to configuration once that's in.

File: git_lint_branch/linter_output.py (hunk 1 above)
Comment (jiviteshjain) [reply git_lint_branch/linter_output.py:3]: Done. This is the one thing that our linter can't programmatically detect :p

**Review by juped — APPROVED** [review]
